e2e/js/js-cloud-server/project.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "e2e-js-cloud-server",
+    "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+    "sourceRoot": "e2e/nextjs/app-router/src",
+    "targets": {
+        "serve": {
+            "executor": "nx:run-commands",
+            "options": {
+                "command": "yarn run -T wrangler dev",
+                "cwd": "examples/js-cloud-server/cloudflare-worker"
+            }
+        },
+        "e2e": {
+            "executor": "@nrwl/jest:jest",
+            "options": {
```

Comment:
maybe add options to start the cloudflare worker here instead of in the yml file of the e2e tests

Reply:
I tried, and it does not seem to work, still getting `fetch failed` any reason against putting it in the workflow file?

Reply:
same reason as the other comment, it's unnecessarily building the sdk and running the worker before it determines whether or not it even needs to